### DISCUSSION TITLE
Unable to create new users, api_not_found in logs

### DISCUSSION THREAD

**Discussion by maurice-audin** [dani-garcia/vaultwarden#2979] [Q&A]
### Subject of the issue

Cannot create new users, errors api_not_found in logs

### Deployment environment

* Vaultwarden version: v1.26.0
* Web-vault version: v2022.11.2
* Running within Docker: false (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: false (X-Forwarded-For)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.35.4
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Steps to reproduce

Build last releases for vaultwarden and web vault from sources

### Expected behaviour

User creation

### Actual behaviour

Red popup (see screenshot) and this in logs:

```
[request][INFO] GET /api/devices/knowndevice/[MAIL]/[ID]
[_][WARN] Response was `None`.
[_][WARN] Responding with registered (api_not_found) /api 404 catcher.
[response][INFO] (web_files) GET /<p..> [10] => 404 Not Found
```

### Troubleshooting data

![screenshot](https://user-images.githubusercontent.com/851854/207306445-fd8d6a20-01b0-4929-b1ae-29d630860875.png)

**Comment by BlackDex** [comment]
You are using an incompatible Vaultwarden server.
Please use the `testing` tagged images which contain the latest and compatible web-vault already instead of merging them your self.

**Comment by maurice-audin** [comment]
I reverted the webvault to v2022.10.2. We build vaultwarden and webvault from sources and run it without docker. So we would prefer to use tagged releases for both vaultwarden and webvault rather than `testing` docker image. I strongly agree with the idea of a compatibility matrix (https://github.com/dani-garcia/vaultwarden/discussions/2944#discussioncomment-4412703)

**Reply by BlackDex** [reply]
If you use tagged releases, you can check the `Dockerfile.j2` for the web-vault version there for the time being.

**Reply by maurice-audin** [reply]
Great !

I can use :

```
$ export VAULTWARDEN_VERSION=1.26.0; curl -s "https://raw.githubusercontent.com/dani-garcia/vaultwarden/${VAULTWARDEN_VERSION}/docker/Dockerfile.j2" | grep '{% set vault_version = ".*" %}' | sed 's/.*"\(.*\)".*/\1/'
v2022.10.0
```

The same intel in bw_web_builds repository would be perfect (releases v2022.10.1 and v2022.10.2 seems ok with vaultwarden 1.26.0).

**Reply by BlackDex** [reply]
Mostly the y updates x.x.y are fine to use. As long as the x.x part matches.
